src/components/ShippingDetails/__tests__/__snapshots__/ShippingDetails.test.js.snap

```diff
@@ -182,7 +182,7 @@ Array [
         <span
           className="c2"
         >
-          Fri, Apr 19 - Mon, Apr 22
+          Tue, Apr 23 - Fri, Apr 26
```

Comment:
red flag! This is a bad test 🐛, `ShippingDetails` component there is an external function `formatDate` that is not being mocked. It is not related with this PR I know.

Reply:
I fixed in in previous PR :)

Reply:
So maybe is better to remove this file from the changes 👍